### PR TITLE
Ghost vr

### PR DESCRIPTION
**Pull request by Crystalwarrior**
ghost vr town fun time

### CODE REVIEW

File: code/mob.dm [hunk 1]
```diff
@@ -2080,7 +2080,7 @@
 		if (M.client && M.client.deadchatoff)
 			continue
 		//admins can toggle deadchat on and off. This is a proc in admin.dm and is only give to Administrators and above
-		if (M.stat == 2 || istype(M, /mob/wraith) || (M.client && M.client.holder && M.client.deadchat && !M.client.player_mode))
+		if (M.stat == 2 || istype(M, /mob/wraith) || (istype(M, /mob/living/carbon/human/virtual) && M:isghost) || (M.client && M.client.holder && M.client.deadchat && !M.client.player_mode))
```
Comment (Iamgoofball) [inline code/mob.dm:2083]: >:

Comment (Crystalwarrior) [reply code/mob.dm:2083]: I mean shit what's a good alternative I haven't coded in byond for a year

Comment (ZeWaka) [reply code/mob.dm:2083]: We have a compiler proc isdead(x) that will be substituted.

File: code/WorkInProgress/simroom.dm [hunk 2]
```diff
@@ -259,6 +259,10 @@
 	set src in oview(1)
 	set category = "Local"
 
+	if(isobserver(usr)) //let ghosts have their VR fun!
+		Station_VNet.Enter_Vspace(usr, src, src:network)
```
Comment (ghost) [inline code/WorkInProgress/simroom.dm:263]: colons are bad

**Pull request closed without merging**